shell/platform/darwin/ios/framework/Source/FlutterEngine_Internal.h

```diff
@@ -49,6 +51,16 @@
 - (flutter::PlatformViewIOS*)iosPlatformView;
 
 - (void)waitForFirstFrame:(NSTimeInterval)timeout callback:(void (^)(BOOL didTimeout))callback;
+
+/**
+ * Creates one FlutterEngine from another, sharing components between them.
+ * This results in a faster creation time and a smaller memory footprint engine.
+ *
+ * @see http://flutter.dev/go/multiple-engines
```

Comment:
Lets make these all code comments

Reply:
I don't think that is appropriate.  Docstrings are the proper place for documenting what a method does.

Reply:
It's not documentation though. i.e. we don't want our users to actually read the doc in a year. It's good context for engineers and maintainers to figure out the project scope and technical decisions.

Reply:
You talking about the link to the design doc?  I've removed that.